app/views/shared/_supporting_information.html.erb

```diff
@@ -115,6 +112,38 @@
       <p class='govuk-body'>Consultation is not applicable for proposed permitted development.</p>
     </div>
   </div>
+  <div class="govuk-accordion__section">
+    <div class="govuk-accordion__section-header">
+      <h2 class="govuk-accordion__section-heading">
+        <button class="govuk-accordion__section-button" id="audit-log">
+          Audit log
+        </button>
+        <span class="govuk-accordion__icon" aria-hidden="true"></span>
+      </h2>
+    </div>
+    <div class="govuk-accordion__section-content" aria-labelledby="audit-log">
+      <% if @audit.present? %>
```

Comment:
that's a lot of logic in the view. Also can we call it `@last_audit`, on its own it's hard to figure out which special audit it is